scripts/export.py

```
@@ -487,6 +487,43 @@ def create_cvs_file(header, rows, filename, output_path):
+:param controls_list: list of controls to augment
+:param frameworks_list: list of frameworks to extract control-framework relationships from
+"""
+def add_framework_references_to_controls(controls_list: list, frameworks_list: list):
+    logging.info("Adding framework references to controls")
+    
+    # Build a mapping from controlID to list of framework names
+    control_to_frameworks = {}
+    for framework in frameworks_list:
+        framework_name = framework.get('name')
+        if not framework_name:
+            logging.warning(f"Framework without name found, skipping")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)